### ISSUE TITLE
Document how arrays of literals are written

### ISSUE THREAD

**Issue by jpmckinney** [kindly/flatterer#18]
```json
[
    {
        "foo": [
            1,
            2
        ]
    }
]
```

yields:

```
_link,_link_table,foo
1,1,"[1,2]"
```

This approach should be documented, e.g. "If a JSON value is an array without any object elements, then it is written as compact JSON."

**Comment by kindly** [comment]
Yep the output format needs a lot of documentation work.
#5

**Comment by kindly** [comment]
Cosing not fixed, but will be part of #5.
